apps/src/calc/expressionNode.js

```diff
@@ -166,7 +166,11 @@ ExpressionNode.prototype.evaluate = function (globalMapping, localMapping) {
       // We're calling a new function, so it gets a new local scope.
       var newLocalMapping = {};
       functionDef.variables.forEach(function (variable, index) {
-        var childVal = this.getChildValue(index);
+        var evaluation = this.children_[index].evaluate(globalMapping, localMapping);
```

Comment:
Just for my own understanding, what's the difference between using `getChildValue` vs. evaluating the child at that index? Is it that the evaluated version will get processed much further?